### PR TITLE
Add simplest integration test for simulate_prod

### PR DESCRIPTION
**Pull request by orelgueta**
Add integration test for simulate_prod and fix that the output directory is not the model directory.

### CODE REVIEW

File: tests/integration_tests/test_applications.py [hunk 1]
```diff
@@ -559,9 +559,57 @@
...
+            "0",
+            "--run",
+            "1",
+            "--nshow",
+            "10",
+            "--data_directory",
+            "TESTOUTPUTDIR/",
+        ]
+    ],
+    "simulate_prod::gamma_20_deg_pack_for_grid": [
```
Comment (GernotMaier) [inline tests/integration_tests/test_applications.py:591]: I don't see what is different in simulate_prod::gamma_20_deg_pack_for_grid compared to "simulate_prod::gamma_20_deg. It is only the `--run` option - does this need to be tested?

**Review by GernotMaier — COMMENTED** [review]
One important question regarding the two tests.

File: tests/integration_tests/test_applications.py (hunk 1 above)
Comment (orelgueta) [reply tests/integration_tests/test_applications.py:591]: You are right, while changing options back and forth I forgot to put back the `"--pack_for_grid_register",` option for the second test. I added it now.

Comment (GernotMaier) [reply tests/integration_tests/test_applications.py:591]: Good!

**Review by GernotMaier — APPROVED** [review]

**Pull request merged**